.github/workflows/operate-ci.yml

```diff
@@ -9,17 +9,27 @@ on:
       - 'main'
       - 'stable/**'
       - 'release/**'
-    paths-ignore:
-      - '.github/workflows/zeebe-*'
-      - 'dist/**'
-      - 'zeebe/**'
-      - 'spring-boot-starter-camunda-sdk/**'
+    paths:
+      - '.ci/**'
+      - '.github/workflows/actions/**'
+      - '.github/workflows/operate-*'
+      - 'bom/*'
+      - 'distro/**'
+      - 'operate.Dockerfile'
+      - 'operate/**'
+      - 'parent/*'
+      - 'pom.xml'
```

Comment:
Is it possible to exclude `operate/client/**` here? At least it's not necessary to run the backend tests on frontend only changes. Do you agree @camunda/operate-backend ?

Reply:
ci-build step was required to build a docker image to be used for the preview environment , but now there is no preview in the mono-repo :/

Reply:
For the tests there is a condition in https://github.com/camunda/zeebe/blob/6f4b96ed73cfd4d6e2fa4e19967f8656e90ca32a/.github/workflows/operate-ci-test-reusable.yml#L44 which skips the tests, my bad...

Reply:
Then it's okay to keep the build step in for now. I'm hoping to get preview environments back soon (pretty please :smile:)

Reply:
> to get preview environments back soon

It is on the TODO list of features to get for sure. Maybe makes sense to wait for single jar and single Docker image so that the preview environments can directly use those instead of replicating the old way.